src/module/s3.js

```diff
@@ -51,9 +56,14 @@ module.exports.S3 = ({ call }) => {
         ...(continuationToken === undefined && startAfter !== undefined ? { StartAfter: startAfter } : {}),
         ...(continuationToken === undefined ? {} : { ContinuationToken: continuationToken })
       });
-      result.push(...response.Contents);
-      continuationToken = response.NextContinuationToken;
-      isTruncated = response.IsTruncated;
+      if (stopAfter !== null && response.Contents[response.Contents.length - 1].Key >= stopAfter) {
+        result.push(...response.Contents.slice(0, response.Contents.findIndex((e) => e.Key > stopAfter)));
+        isTruncated = false;
```

Comment:
set to `false` to stop paging. Should we also set `continuationToken` to undefined

Reply:
yeah I think so